packages/panels/src/components/action-menu-button/ActionMenuButton.tsx

```diff
@@ -93,11 +93,7 @@ export const ActionMenuButton: React.FC<ActionMenuButtonProps> = ({
         onClickOutside={close}
         placement={placement}
         content={
-          isOpen && (
-            <FocusScope contain>
-              <Box>{renderItems(close)}</Box>
-            </FocusScope>
-          )
+          isOpen && <ActionMenu trapFocus>{renderItems(close)}</ActionMenu>
```

Comment:
We must decide if we want renderItems to render ActionMenu or not. If not, we must make it possible to set the ActionMenu props through ActionMenuButton (width, etc..)

Reply:
Yeah, we've been through this problem before :(
Same with calendar inputs, we pass `calendarProps` which is not a very elegant solution.

Fortunately, `ActionMenu` has a lot less relevant props. Maybe `width` is enough? Or is it even needed when placed inside a `Popover`?

Reply:
Regardless, this doesn't change the current API (`renderItems`-prop), so we can discuss this when the needed.